src/main/java/com/github/legoatoom/connectiblechains/chain/ChainType.java

```diff
@@ -0,0 +1,18 @@
+package com.github.legoatoom.connectiblechains.chain;
+
+import com.github.legoatoom.connectiblechains.client.ClientInitializer;
+import net.minecraft.item.Item;
+import net.minecraft.util.Identifier;
+
+/**
+ * The 'material' of a chain
+ */
+public record ChainType(Item item) {
+    public Identifier getKnotTexture() {
+        return ClientInitializer.getInstance().textureManager.getKnotTexture(ChainTypesRegistry.REGISTRY.getId(this));
+    }
+
+    public Identifier getChainTexture() {
+        return ClientInitializer.getInstance().textureManager.getChainTexture(ChainTypesRegistry.REGISTRY.getId(this));
+    }
+}
```

Comment:
This file tries to access client classes but is not inside the client directory.
Can this not cause issues as the server does not contain such classes?